.github/workflows/perf.yml

```diff
@@ -18,12 +18,12 @@ jobs:
       - name: Run benchmark
         run: make gobenchmark
 
-      - name: Store benchmark result
-        uses: benchmark-action/github-action-benchmark@v1
-        with:
-          tool: 'go'
-          output-file-path: benchmarks.txt
-          gh-pages-branch: gh-pages
-          auto-push: true
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          benchmark-data-dir-path: "docs/dev/bench"
+      #- name: Store benchmark result
```

Comment:
Maybe add a comment why it's commented? Or just remove?

Reply:
Good point, updated to include the comment.